scripts/bootstrap-vault.sh

```diff
@@ -0,0 +1,51 @@
+# Init vault and save keys
+kubectl exec vault-0 -- vault operator init -key-shares=1 -key-threshold=1 -format=json > cluster-keys.json
+
+# Store key and unseal vault
+VAULT_UNSEAL_KEY=$(cat cluster-keys.json | jq -r '.unseal_keys_b64[]')
```

Comment:
Generally this would be considered an unnecessary use of cat. Try this:
```suggestion
VAULT_UNSEAL_KEY=$(jq -r '.unseal_keys_b64[]' < cluster-keys.json)
```

Reply:
See some of your other uses of cat also below.

Reply:
In fact, I'm pretty sure this works, too, and is probably better:
```suggestion
VAULT_UNSEAL_KEY=$(jq -r '.unseal_keys_b64[]' cluster-keys.json)
```

Reply:
Will do.